docs/Resonance_Compensation.md

```diff
@@ -45,31 +44,25 @@ Slice the ringing test model, which can be found in
 
 First, measure the **ringing frequency**.
 
-1. Increase `max_accel` and `max_accel_to_decel` parameters in your
-   `printer.cfg` to 7000. Note that this is only needed for tuning, and more
-   proper value will be selected in the corresponding
-   [section](#selecting-max_accel).
-2. If `square_corner_velocity` parameter was changed, revert it back to 5.0.
-   It is not advised to increase it when using the input shaper because it can
-   cause more smoothing in parts - it is better to use higher acceleration
-   value instead.
-3. Restart the firmware: `RESTART`.
-4. Disable Pressure Advance: `SET_PRESSURE_ADVANCE ADVANCE=0`.
-5. If you have already added `[input_shaper]` section to the printer.cfg,
+1. Disable `max_accel_to_decel` and use the default value of
+   `square_corner_velocity` by issuing the following command:
+   `SET_VELOCITY_LIMIT ACCEL_TO_DECEL=7000 SQUARE_CORNER_VELOCITY=5`
+2. Disable Pressure Advance: `SET_PRESSURE_ADVANCE ADVANCE=0`
+3. If you have already added `[input_shaper]` section to the printer.cfg,
    execute `SET_INPUT_SHAPER SHAPER_FREQ_X=0 SHAPER_FREQ_Y=0` command. If you
    get "Unknown command" error, you can safely ignore it at this point and
    continue with the measurements.
-6. Execute the command
-   `TUNING_TOWER COMMAND=SET_VELOCITY_LIMIT PARAMETER=ACCEL START=1250 FACTOR=100 BAND=5`.
+4. Execute the command:
+   `TUNING_TOWER COMMAND=SET_VELOCITY_LIMIT PARAMETER=ACCEL START=1250 FACTOR=100 BAND=5`
```

Comment:
I see that TUNING_TOWER has now the STEP_DELTA and STEP_HEIGHT parameters. Would it make sense to update this command here and below in other instances to use them instead? I think we get countless questions on discord where people are confused and ask if the command is wrong. It will read

`TUNING_TOWER COMMAND=SET_VELOCITY_LIMIT PARAMETER=ACCEL START=1500 STEP_DELTA=100 STEP_HEIGHT=5`

if I am not mistaken